src/watch/utils.ts

```diff
@@ -29,3 +31,10 @@ export const debounce = <T extends (this: unknown, ...args: any[]) => void>(
 		);
 	} as T;
 };
+
+export const resolveGlobPattern = (pattern: string): string => {
+	if (path.isAbsolute(pattern)) {
+		return normalizePath(pattern);
+	}
+	return normalizePath(path.join(process.cwd(), pattern));
+};
```

Comment:
This only seems used to normalize the path slashes in user provided paths—do they need _need_ to be normalized?

Reply:
Without normalization, Windows style separators break the glob logic. Normalizing the paths make all our paths we use consistent and easier to handle (e.g. we can match our globs based on Unix-style separators instead of having to figure out which it is)